h/groups/models.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+import sqlalchemy as sa
+from sqlalchemy.orm import exc
+
+from h.db import Base
+from h.textutil import slugify
+
+
+class Group(Base):
+    __tablename__ = 'group'
+
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+    name = sa.Column(sa.Unicode(100), nullable=False)
```

Comment:
Honestly, I just figured that

    East Louisiana State Troopers Chinchilla Brevicaudata Appreciation Society - East Baton Rouge Branch

Was more than long enough for a group name.